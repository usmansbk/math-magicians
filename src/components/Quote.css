@import url("https://fonts.googleapis.com/css2?family=Playfair+Display:ital@1&display=swap");

.quote-section {
  display: flex;
  flex-grow: 1;
  flex-direction: column;
  gap: 40px;
  padding: 10%;
  background-image: url(../imgs/quote.png);
  background-repeat: no-repeat;
  background-size: 160px;
}

.quote {
  font-size: 30px;
  color: #212121;
  font-family: "Playfair Display", serif;
}

.line {
  height: 4px;
  background-color: #ff7f50;
  width: 30%;
  align-self: center;
}
