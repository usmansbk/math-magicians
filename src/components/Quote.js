import './Quote.css';

const quote = 'Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. - William Paul Thurston';

const Quote = () => (
  <section className="quote-section">
    <div className="line" />
    <h5 className="quote">{quote}</h5>
  </section>
);

export default Quote;
