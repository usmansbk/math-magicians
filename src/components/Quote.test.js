import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quote';

test('expect quote page to match working snapshot', () => {
  const quotePage = renderer.create(<Quote />).toJSON();
  expect(quotePage).toMatchSnapshot();
});
