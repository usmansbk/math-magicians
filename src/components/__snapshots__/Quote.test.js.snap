// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect quote page to match working snapshot 1`] = `
<section
  className="quote-section"
>
  <div
    className="line"
  />
  <h5
    className="quote"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. - William Paul Thurston
  </h5>
</section>
`;
